hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java

```diff
@@ -148,7 +148,7 @@ public void fallbacksApplyOnError() {
   @Test
   public void errorInFallbackHasExpectedBehavior() {
     thrown.expect(HystrixRuntimeException.class);
-    thrown.expectMessage("contributors failed and fallback failed.");
+    thrown.expectMessage("feign#public abstract java.util.List feign.hystrix.HystrixBuilderTest$GitHub.contributors(java.lang.String,java.lang.String) failed and fallback failed.");
```

Comment:
general question is why we'd use something uglier and without any semantic
gain in uniqueness, and also not the convention of feign, when we have a
choice not to?

On Wed, Aug 17, 2016 at 4:10 AM, Ryan Baxter <notifications@github.com>
wrote:

> In hystrix/src/test/java/feign/hystrix/HystrixBuilderTest.java
> <https://github.com/OpenFeign/feign/pull/444#discussion_r75010075>:
>
> > @@ -148,7 +148,7 @@ public void fallbacksApplyOnError() {
> >    @Test
> >    public void errorInFallbackHasExpectedBehavior() {
> >      thrown.expect(HystrixRuntimeException.class);
> > -    thrown.expectMessage("contributors failed and fallback failed.");
> > +    thrown.expectMessage("feign#public abstract java.util.List feign.hystrix.HystrixBuilderTest$GitHub.contributors(java.lang.String,java.lang.String) failed and fallback failed.");
>
> So it is not ideal, but in my opinion this is a separate issue from the
> one we are trying to solve
> [image: image]
> <https://cloud.githubusercontent.com/assets/524254/17714024/de6541cc-63cb-11e6-8e4a-4a6a83b2e19b.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenFeign/feign/pull/444/files/0dadae29a8db6edf595fad0d35b3346472a5243d#r75010075>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAD6107T4vCm7_7rJ6igFh45cO8x8bPCks5qghkZgaJpZM4JlhA6>
> .
>